interfaces/builtin/kernel_module.go

```diff
@@ -0,0 +1,48 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+var kernelModuleConnectedPlugAppArmor = `
+# Description: Allow insertion of modules.
+
+  capability sys_module,
+`
+
+var kernelModuleConnectedPlugSecComp = `
+# Description: Allow insertion of modules.
+
+finit_module
+delete_module
```

Comment:
I see in the man page that query_module is deprecated, yes, so fine to leave that out. Thanks for adding init_module and @{PROC}/modules.